spec/request/api/v1/users/show_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+describe 'GET api/v1/user/id', type: :request do
+  let!(:user) { create(:user, :confirmed) }
+
+  subject { get api_v1_user_path(user), headers: auth_header }
+
+  context 'when the request is succesful and user has avatar' do
+    before do
```

Comment:
you can use the `trait` created in the user factory instead of attaching the avatar before each test